source/instructors/authoring/assessments/student-submission.rst

```diff
@@ -3,51 +3,76 @@
   
 .. _student-submission:
 
-Student submission options
+Student Submission Options
 ==========================
-There are two important settings that control
+There are two important settings to control:
```

Comment:
The re-wording here kind of changes the meaning. "two important settings to" implies that you have to control them whereas the actuality is that they "can" be controlled if you want to do so. Maybe just change it to something like "You can control the following things:"
- How students submit individual questions.
- How assignments are marked as completed. [my comment - because they can "automatically" be marked as complete - out of student control - at a specific time]